### DISCUSSION TITLE
bundle-analyzer, The term 'ANALYZE=true' is not recognized as the name of a cmdlet, function, script file, or operable program.

### DISCUSSION THREAD

**Discussion by bayswaterpc** [vercel/next.js#29107] [Help] [answered]
### What version of Next.js are you using?

10.2

### What version of Node.js are you using?

14.7.1

### What browser are you using?

Chrome

### What operating system are you using?

Windows

### How are you deploying your application?

Vercel

### Describe the Bug

I'm trying to get the bundle analyzer working with a blog I'm working on.  I'm following the steps listed at the [@next/bundle-analyzer](https://www.npmjs.com/package/@next/bundle-analyzer)

When i run `ANALYZE=true yarn build`

I get the following error message

ANALYZE=true : The term 'ANALYZE=true' is not recognized as the name of a    
cmdlet, function, script file, or operable program. Check the spelling of    
the name, or if a path was included, verify that the path is correct and     
try again.
At line:1 char:1
+ ANALYZE=true yarn build
+ ~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (ANALYZE=true:String) [], Com  
   mandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

This is most likely a windows config issue as it seemed similar to this  [stack overflow thread](https://stackoverflow.com/questions/35810172/webpack-is-not-recognized-as-a-internal-or-external-command-operable-program-or) which had fixed issues I had with another project and build analyzer.

If any windows dev out there has tips would be much appreciated.  Thanks.

### Expected Behavior

The bundle analyzer should popup.

### To Reproduce

You install this [project](https://github.com/bayswaterpc/bayswaterpc-superplate-blog) and try running `ANALYZE=true yarn build` on a windows machine, I assume it will be the same

**Comment by ijjk** [comment, marked as answer]
Hi, this is due to how environment variables are handled in the Windows command prompt, you should be able to use `set ANALYZE=true` and then `yarn build` instead or you can use `npx cross-env ANALYZE=true yarn build`.

**Reply by Sujood-Qureshi** [reply]
do you have any solution for ANALYZE=true npm run build actually i am using npm and its giving me the same error

**Reply by gupta82anish** [reply]
you can `npx cross-env ANALYZE=true npm run build`. It should work

**Comment by bayswaterpc** [comment]
The `npx cross-env ANALYZE=true yarn build` fixed it for me, thanks so much!

**Comment by arunkatoch-dev** [comment]
This code worked for me
next.config file:
/** @type {import('next').NextConfig} */

import bundleAnalyzer from "@next/bundle-analyzer";

const withBundleAnalyzer = bundleAnalyzer({
  enabled: process.env.ANALYZE === "true",
});

export default withBundleAnalyzer({
  // Other Next.js configuration options here

});

run command:
npx cross-env ANALYZE=true yarn build
